ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -485,15 +485,15 @@ export default class TaskRunsCard extends Component {
                 onClick={this.handleNext}
                 disabled={run ? selectedRunId === runs.length - 1 : true}>
                 Next
-                <ChevronRightIcon />
+                <ChevronRightIcon color="rgb(149, 27, 27)" />
```

Comment:
I think adding custom `RGB Codes` is not an optimal solution for the following. You can check other icons implementation and then do the same.

Reply:
Yes, it is not an optimal solution, but the other icons still differ in the color only, it is a minor problem related to the icons not visible in the light/dark theme. So this can be handled by custom RGB codes.